homeassistant/components/switch/wemo.py

```diff
@@ -110,11 +111,30 @@ def device_state_attributes(self):
         if self.insight_params or (self.coffeemaker_mode is not None):
             attr[ATTR_CURRENT_STATE_DETAIL] = self.detail_state
 
+        if self.insight_params:
+            attr['on_latest_time'] = \
```

Comment:
The only one you'd need to rename is the threshold in `mw`.

Perhaps switch it to `W’ and break out the attributes in constants. Then nothing will need to be changed if my PR gets merged.